web_notify/README.rst

```diff
@@ -64,7 +64,7 @@ or
    :alt: Sample notifications
 
 You can test the behaviour of the notifications by installing this module in a demo database.
-   Click in your user logo in the top right corner of the screen then click "Preferences".
```

Comment:
this is duplicated below, keep instructions in USAGE only

Reply:
What's duplicated? README.rst is the result of the new generator that takes content from ```readme/``` folder...

Reply:
hmmm sorry, I mis-read the path above the content! :stuck_out_tongue_closed_eyes: